docs/video/index.md

```diff
@@ -5,27 +5,25 @@ icon: material/video-box
 
-Video editors such as [Vegas Pro](/placeholder/) and [Adobe Premiere Pro](/placeholder/) have these features built-in. However, we recommend using seperate programs such as [blur](/placeholder/) first to pre-render the videos, therefore you don't have any lag in your video editor as well as having much more customisation.
+Video editors such as [Vegas Pro](/placeholder/) and [Adobe Premiere Pro](/placeholder/) have these features built-in. However, we recommend using seperate programs such as blur or smoothie first to pre-render the videos, since they're faster and more configurable with how they handle frame blending. Using them will also decrease preview lag in editors.
 
 ### :material-select-multiple: Interpolation
 
-Video interpolation is a video processing technique that creates new frames between existing ones, effectively increasing the video's frames per second (FPS) using algorithms or AI.
+Video interpolation is a video processing technique that creates new frames between existing ones, effectively increasing the video's framerate. This is done using algorithms (such as SVP) or AI models (such as RIFE).
 
-[RIFE](https://github.com/megvii-research/ECCV2022-RIFE) and the [SVP](https://www.svp-team.com) algorithm are commonly implemented into frame blending applications to interpolate.
+However, **interpolation is not and never will be perfect**. The good news is that it can do a very convincing job of appearing perfect with higher framerate inputs. Higher is better, and lower is always worse, as long as your recording framerate doesn't exceed your in-game framerate and you don't have too many dropped or duplicated frames (which can be seen in skipped frames in OBS' stats dock).
 
-It generates frames from the data it has, and the higher the input FPS, the smaller the window between the two points are, which means more FPS the better.
+### :material-file-video-outline: Video Encoding/Decoding
 
-### :material-file-video-outline: Video encoding and decoding
+Video encoding is the process of converting raw video into a digital codec, with decoding converting videos into a format that can be displayed and viewed on a screen.
 
-Video encoding is the process of converting one format (e.g. the raw visual data captured during recording) into a digital codec, with decoding converting videos into a format that can be displayed and viewed on a screen.
-
-A video codec is software or hardware that compresses and decompresses digital video to make file sizes smaller and make storage and distribution of videos easier.
\ No newline at end of file
+A video codec is software or hardware that can encode and decode videos. The most common video codec is H.264 (aka AVC), which is what is most commonly used and references here.
```

Comment:
'references'

Reply:
![image](https://github.com/couleur-tweak-tips/CTT/assets/100049254/dac7982f-4fef-4398-9116-152b2911a55c)

Reply:
this is unacceptable, please fix asap